ci-operator/step-registry/shiftstack/bugwatcher/shiftstack-bugwatcher-commands.sh

```diff
@@ -10,4 +10,4 @@ export BUGZILLA_API_KEY
 export SLACK_HOOK
 export TEAM_MEMBERS
 
-python ./main.py &> "${ARTIFACT_DIR}/bugwatcher.log"
+python ./main.py
```

Comment:
Boo!

Reply:
It’s there! https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-shiftstack-bugwatcher-main-bugwatcher/1402749947467534336/artifacts/bugwatcher/shiftstack-bugwatcher/build-log.txt